apps/test/unit/util/formatValidationTest.js

```diff
@@ -91,12 +92,13 @@ describe("formatValidation", () => {
     it("Rejects invalid percentages", () => {
       [
         "-1",
-        "101",
+        "100.5",
+        "100.5%",
```

Comment:
I wonder if we should be friendly about this one and attempt to strip a trailing `%` sign?

Reply:
Never mind. Of course, 100.5 is not a valid percentage.